modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java

```diff
@@ -1290,7 +1300,15 @@ public void visit(BLangRecordLiteral recordLiteral) {
     }
 
     public void visit(BLangTernaryExpr ternaryExpr) {
-        /* ignore */
+        this.genNode(ternaryExpr.expr, this.env);
```

Comment:
Please move this code to expression section. We need to move all the remaining methods as well